provider/postgis/util.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strconv"
 	"strings"
 
+	ent "github.com/SharperShape/entity-id"
```

Comment:
This is the old library. The new one is `github.com/SharperShape/go-service-common/ent`. We can switch to that one.

Reply:
Nah, let's leave it as is for now.

Reply:
We should not be relying on either here, tbh.

Reply:
@kszafran Which repos rely on the old id implementation library? Can we just get rid of it?

Reply:
My search shows only this one.

Reply:
Yep, only this one. The `entity-id` repo is already archived.